packages/pie-inline-choice/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "pie-inline-choice",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/index.js",
```

Comment:
you'll need to add: 
```
"publishConfig": {
    "access": "public"
  }
```
So that the scoped package will publish properly

Reply:
Done